packages/core/src/routes/user.ts

```diff
@@ -66,4 +66,31 @@ export default function userRoutes<T extends AnonymousRouter>(router: T) {
       return next();
     }
   );
+
+  /**
+   * User Custom Data
+   */
+  router.patch(
+    '/user/:userId/custom-data',
```

Comment:
should be merged into `adminUser` route? @wangsijie @simeng-li

Reply:
yes, my fault.